### PR TITLE
Tobias/bugfix

### PR DESCRIPTION
**Pull request by TobiasKjernell**

### CODE REVIEW

**Review by andrewkalumba — APPROVED** [review]
Looks great. And you can proceed

**Review by Svpriyaa2808 — APPROVED** [review]
Looks good

File: styles/card.css [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
-    left: 50%;
-    transform: translate(-50%, -50%);
-
-    @media screen and (min-width:800){
-    
-    }
+    width: 30rem;
+    margin: 0 auto;
+    @media screen and (max-width:900){
+
```
Comment (VaishaliGovindaraj) [inline styles/card.css:7]: please remove whitespace

Comment (TobiasKjernell) [reply styles/card.css:7]: good eye

**Review by VaishaliGovindaraj — APPROVED** [review]
everything looks good

**Review by ranjithavelusamy — APPROVED** [review]
Everthing looks good

**Pull request merged**